selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -1484,6 +1494,7 @@ private void waitTextInJavaDoc(String expectedText) {
    * @return true - if {@code expectedText} is present in javadoc body, false - if not
    */
   public boolean waitAndCheckTextPresenceInJavaDoc(String expectedText) {
+    seleniumWebDriverHelper.moveCursorTo(By.xpath(JAVA_DOC_POPUP));
```

Comment:
Is Javadoc still a thing? Shouldn't this be some generic language server popup?

Reply:
renamed to `TEXT_VIEW_TOOLTIP`